qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -6,4 +6,4 @@ webdriver=chrome
 login.auto=true
 headless=true
 webdriver.gecko.driver=firefoxdriver/linux/geckodriver
-includes.csrftoken=true
+includes.csrftoken=false
```

Comment:
I think this property should be `true ` for tests to be able to login into the instance.